muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -66,11 +73,46 @@ def check_multilabel_predictions(clf, X, y):
         "multilabel classifier is outputting distributions"
 
 
+def check_multilabel_predictions_na(clf, X, y):
+    predicted = clf.fit(X, y).predict(X)
+    # change nans to zero to 'mask' those values
+    is_na = (y == -1)
+    predicted[is_na] = 0
+    y_temp = y.copy()
+    y_temp[is_na] = 0
+
+    assert_equal(predicted.shape, y_temp.shape)
+    assert_array_equal(predicted, y_temp)
+
+    probabilities = clf.predict_proba(X)
+    probabilities[is_na] = 0
+    assert_equal(probabilities.shape, y_temp.shape)
+    assert_array_equal((probabilities >= 0.5).astype(np.int), predicted)
```

Comment:
I think so. `Probabilities` is being compared to `predicted`, which is being masked.